Add sectionabout aa1 to master

petschna opened the pull request:
**Checklist:**

 <!--- Make sure your PR is documented and tested before submission. Put an `x` in all the boxes that apply: -->
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?


**Type of Changes:**

 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)


**What is the current behavior? (link to any open issues here)**
- Currently there is no section about, so there is also no explanation for what erebos actually is. #5


**What is the new behavior (if this is a feature change)?**
- Now there is a section about which explains what erebos is and what we stand for. Closes #5

petschna committed:
Add component SectionAbout

The organism SectionAbout has been
added to the environment.

petute reviewed (approved):
Should be ok to merge.

petschna committed:
Add AboutPage to PageRoot

The SectionAbout has
been added to the PageRoot.js.

petschna committed:
Remove entry global Navbar and global Footer

The global Navbar and the global Footer entry has been
removed from the App,js and index.js, because the
corresponding objects were not pushed.

petschna committed:
Add SectionAbout background image

The background image of the SectionAbout
has been added to the environment.

kleberbaum reviewed (approved):
Should be ok to merge.

Aichnerc reviewed (approved):
LGTM 🦐
